### ISSUE TITLE
Creatures should always keep the job you assign them to

### ISSUE THREAD

**Issue by Loobinex** [OpenDungeons/OpenDungeons#1231]
In this game it feels you have very little control of what your creatures are doing.  Part of this is because when you assign a creature to work a room, they will just quit and do something else as soon as they come back from eating/sleeping/gold.

This should not be the case, once assigned to a job a creature should keep returning there unless he has a valid reason not to.

I've attached a save from git20160725 to see cultists getting back to research after I have told them several times to start training.
[2016-07-27_000715-SK-Angel.zip](https://github.com/OpenDungeons/OpenDungeons/files/384999/2016-07-27_000715-SK-Angel.zip)

**Comment by hwoarangmy** [comment]
Yes, that's the current behaviour.

**Comment by Loobinex** [comment]
What is? Going back to the old job, or changing jobs after eating?
If the former, it is not working and I've got the saved game to prove it. (and the replay as well, see issue 1238).

**Comment by hwoarangmy** [comment]
the current behaviour is to not remind the job it is doing when picked up

**Comment by Loobinex** [comment]
Yes, that's fine. But I was not talking about picking it up. Steps to reproduce the issue:
1. Start a map.
2. Research a training room.
3. Assign all cultist you receive to the training room.
4. Observe as cultist will return to research before reaching level 10.

**Comment by hwoarangmy** [comment]
yes, that's what I said. Once they stop working because sleepy/hungry/whatever, they do not remind that they were dropped in some room.

What I want to say is that what you describe is the current behaviour, not a bug. And reminding that the creature has been dropped somewhere would be an improvement, not a fix ;-)
